feedback-front/templates/record_statistics.html

```diff
@@ -0,0 +1,79 @@
+{% extends "layout.html" %}
+{% block body %}
+<link rel="stylesheet" href="https://cdn.rawgit.com/novus/nvd3/v1.8.1/build/nv.d3.css">
+<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js" charset="utf-8"></script> 
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+<script src="{{ url_for('static', filename='js/nv.d3.js') }}"></script>
+<script src="{{ url_for('static', filename='js/charts.js') }}"></script>
+
+<script>
+  
+  // rendering pie chart
+  var data = [{
+    key : "Dataset stats",
+    values : [
+      {% for stat in stats %}
+          {
+              label : "{{ stat.label}}",
+              value : {{ stat.numberOfRecords }}
+          },
+      {% endfor %}
+    ]
+  }];
+  renderBarChart(data, '#chart1'); // function to render bar chart and data as the parameters
+
+  // rendering bar chart
+  var data = [
+        {% for stat in stats %}
+          {
+            label : "{{ stat.label}}",
+            value : {{ stat.numberOfRecords }}
+          },
+        {% endfor %}
+    ];
+    renderPieChart(data, "#chart2");
+</script>
+
+<!-- bar chart divison -->
+<div id="Div1">
+  <div id="chart1">
+    <svg></svg> 
+  </div>
+</div>
+<br>
+
+<!-- pie chart divison -->
+<div id="Div2">
+  <div id="chart2">
+    <svg></svg>
+  </div>
+</div>  
+
+<!-- Javascript for toggling the charts -->
+<script>
+function switchCharts() {
+  if (document.getElementById('Div1')) {
```

Comment:
Don't use generic div names. Name it something that is readable - "barChartDiv" or something.